raphtory/src/python/filter/edge_filter_builders.rs

```diff
@@ -186,6 +192,16 @@ impl PyEdgeFilter {
     fn metadata(name: String) -> MetadataFilterBuilder<EdgeFilter> {
         EdgeFilter::metadata(name)
     }
+
+    #[staticmethod]
+    fn window(py_start: Bound<PyAny>, py_end: Bound<PyAny>) -> PyResult<PyEdgeWindow> {
+        let s = py_into_millis(&py_start)?;
```

Comment:
there should already be support for doing this, it might be TimeIndexComponent

Reply:
Replace this with TimeIndexComponent when merged from master.